src/components/ constants/global.js

```diff
@@ -0,0 +1,8 @@
+export const imgUrl = "https://image.tmdb.org/t/p/w500/";
+export const fakeImg = "http://via.placeholder.com/1080x1580/";
```

Comment:
The image at the most going to be of size 350x374px. Please use appropriate image sizes.